app/views/application/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>New Application</h1>
+<%= form_with url: "/applications", local: true do |f| %>
```

Comment:
While the default form method is POST, it might be helpful to explicitly add that to your form definition just so you are remembering how this form will route. Just a suggestion, not at all required